src/include/elektra.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file
+ *
+ * @brief Elektra High Level API.
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#ifndef ELEKTRA_H
+#define ELEKTRA_H
+
+#include "kdb.h"
+#include "kdbhelper.h"
+#include "kdbtypes.h"
+
+struct _ElektraError
+{
+    int errorNr;
```

Comment:
Yes, we could do something like that (maybe with another function). The idea is that the developer can identify an error-type without parsing the string. Maybe it is not needed, I would say we will see which error-types we will have and can then decide upon that if we keep it or remove it.